platform/android/app/src/main/kotlin/coop/polypoly/polypod/Preferences.kt

```diff
@@ -40,14 +41,29 @@ class Preferences {
             edit.commit()
         }
 
-        fun setBiometricCheck(context: Context, shouldCheck: Boolean) {
+        fun setSecurityDoNotAskAgainCheck(
+            context: Context,
+            shouldCheck: Boolean
+        ) {
             val edit = getPrefs(context).edit()
             edit.putBoolean(biometricCheckKey, shouldCheck)
             edit.commit()
         }
 
-        fun isBiometricCheck(context: Context): Boolean =
-            getPrefs(context).getBoolean(biometricCheckKey, true)
+        fun hasUserConfiguredAuthentication(context: Context): Boolean =
+            getPrefs(context).getBoolean(userConfiguredAuth, false)
+
+        fun setUserConfiguredAuthentication(
+            context: Context,
+            shouldCheck: Boolean
+        ) {
+            val edit = getPrefs(context).edit()
+            edit.putBoolean(userConfiguredAuth, shouldCheck)
+            edit.commit()
+        }
+
+        fun isSecurityDoNotAskAgainEnabled(context: Context): Boolean =
+            getPrefs(context).getBoolean(biometricCheckKey, false)
```

Comment:
Probably this should move up by two functions, below `setSecurityDoNotAskAgainCheck`